### ISSUE TITLE
setupIngressController key not respected

### ISSUE THREAD

**Issue by eysi09** [garden-io/garden#538]
### Current Behavior

Setting `setupIngressController` to false in the `local-kubernetes` provider still set ups the default `nginx` ingress controller. 

### Expected behavior

Ingress controller shouldn't be set if `setupIngressController` is false.

### Reproducible example

In `simple-project`, use the following project config:

```yaml
project:
  name: simple-project
  environments:
    - name: local
      providers:
        - name: local-kubernetes
          setupIngressController: false
```

run

```
kubectl get ingress node-service -o yaml --export --namespace=simple-project | grep ingress.class:
```

and notice that the result is:

```
kubernetes.io/ingress.class: nginx
``` 

### Your environment

`garden version`

Latest `master`, most recent commit hash is `d44f4b3401b0fa54f9e`

`kubectl version`

```
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.0", GitCommit:"ddf47ac13c1a9483ea035a79cd7c10005ff21a6d", GitTreeState:"clean", BuildDate:"2018-12-03T21:04:45Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.0", GitCommit:"ddf47ac13c1a9483ea035a79cd7c10005ff21a6d", GitTreeState:"clean", BuildDate:"2018-12-03T20:56:12Z", GoVersion:"go1.11.2", Compiler:"gc", Platform:"linux/amd64"}
```

`docker version`

```
18.09.1
```

**Comment by edvald** [comment]
I think what may be happening here, is that you leave this option unchanged in other projects, which then _do_ set it up. Not sure how we might handle that. Supporting multiple ingress controllers in a cluster is tricky in general.

The ingress class is incorrectly set at https://github.com/garden-io/garden/blob/be5448381ed1a9c9e1b8808a3279204c8e54ecb2/garden-service/src/plugins/kubernetes/local.ts#L143 though. That should be `config.ingressClass`.

**Comment by edvald** [comment]
Fixed in #553
